test/v0/baldurs_gate_patch_v1_1_4315_international/CMakeLists.txt

```
@@ -0,0 +1,21 @@
+add_test(
+    NAME baldurs_gate_new_compression
+    COMMAND $<TARGET_FILE:unshield> -d extract1 x ${CMAKE_CURRENT_SOURCE_DIR}/data1.cab
+    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/test
+)
+
+add_test(
+    NAME baldurs_gate_old_compression
+    COMMAND $<TARGET_FILE:unshield> -O -d extract2 x ${CMAKE_CURRENT_SOURCE_DIR}/data1.cab
```

```suggestion
    COMMAND $<TARGET_FILE:unshield> -d extract2 x ${CMAKE_CURRENT_SOURCE_DIR}/data1.cab
```

Seems like it should have `-O` if I compare to the shell script

---

There are three test case:

1. testing the new compression
2. testing the old compression
3. validating the MD5 hashes

Depending on the cab files under test, either 1. or 2. is to be expected to fail.
But neither should cause a segmentation fault, which it does in case of Baldurs Gate with the old compression

---

Good point, segfault is not a good behaviour.